packages/ui-shared-lite/utils/storage.ts

```diff
@@ -34,13 +35,13 @@ const storage: BPStorage = {
       const driver = getDriver()
       driver !== 'cookie' ? driver.setItem(key, value) : Cookie.set(key, value)
     } catch (err) {
-      console.error('Error while getting data from storage.', err.message)
+      console.error('Error while setting data into storage.', err.message)
     }
   },
   get: (key: string) => {
     try {
       const driver = getDriver()
-      return driver !== 'cookie' ? driver.getItem(key) : Cookie.get(key)
+      return driver !== 'cookie' ? driver.getItem(key) || undefined : Cookie.get(key)
```

Comment:
what happens if you return undefined

Reply:
It's the caller's job to handle this edge case. By default, `driver.getItem(key)` can return null in case the key is not found in either the sessionStorage or localStorage.

Reply:
Is the caller doing it's job then ?

Reply:
I looked at all the occurrences where it's used and they all handle those cases.